.gitignore

```diff
@@ -3,6 +3,11 @@ node_modules
 
 # IDEA config
 .idea
+*.swp
```

Comment:
what is it?

Reply:
this is a gobal discussion to have. Either remove all editor files, or have them in each project, with a templated .gitignore.

Specificaly vim here.

Reply:
just add a # VIM comment for now :D

Reply:
a bit like idea config for idea .... :D